models/plugin.go

```diff
@@ -378,7 +460,7 @@ func (s *registeredPlugins) GetAll() []*Plugin {
 	for _, unit := range s.data {
 		list = append(list, unit)
 	}
-	return list
+	return s.sortPlugins(list)
```

Comment:
Do we need to sort it every time, can we not sort it at creation time and use the list. ?

Reply:
since repo is a map, either I can maintain sorted keys to return sorted list of plugins or sort every time version is being called (either the case it's once per invocation).
But I see that sort implementation is ugly here. I guess I can maintain sorted list of names. WDYT ?